### ISSUE TITLE
SG Level 2 Subtitles

### ISSUE THREAD

**Issue by jarringjade** [SG-LoGiRL-Subtitles-Team/SG-LoGiRL#2]
The following SG LoGiRL episodes have a Level 2 (**_chat log, cleaned but untimed_**) subtitles:

1. Episode 8
2. Episode 12
3. Episode 17
4. Episode 20
5. Episode 38
6. Episode 40
7. Episode 41
8. Episode 48
9. Episode 51
10. Episode 69
11. Episode 71
12. Episode 74
13. Episode 76
14. Episode 77
15. Episode 80
16. Episode 81

These episodes needs to have the timing fixed.

**Comment by RenRoqs** [comment]
I'm trying in working on Episode 12 but its taking me a lot more time because of school so if you can be patient I'll try and clean it and maybe adjust the timing for it :D

**Comment by cuicuiwa** [comment]
I'm working on Episode 81, a little over halfway done. Aiming to finish it up over the weekend.

**Comment by jarringjade** [comment]
@RenRoqs - Ah, no worries there. I'm also attending school, so I can understand. What we're aiming for is quality over quantity. No need to rush, just take your time to finish that episode.

@cuicuiwa - Are you on Reddit? Do you mind if I ask your username over there? Just so I can account for everybody. Thank you!

**Comment by cuicuiwa** [comment]
@jarringjade Yes, my username on Reddit is CuiCui_wa. I left a comment on the SG LoGiRL Subtitles Improvement GitHub thread as well.

**Comment by RenRoqs** [comment]
@jarringjade also I saw the comment of Swedish (I've forgotten name ;-;) if its hard subbed, how does one and where does one upload it?? Amen pron0ob here

**Comment by jarringjade** [comment]
@RenRoqs - SwedishFukei is nomusician, I believe. Which episode was it?

You can upload files here by dragging and dropping it on the comment box. It attaches to the comment. :)
